packages/big-design/src/components/AccordionGroup/AccordionPanel/AccordionPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import { Text } from '@bigcommerce/big-design';
+import { ExpandLessIcon, ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { HTMLAttributes, memo } from 'react';
+
+import { useUniqueId } from '../../../hooks';
+import { BoxProps } from '../../Box/Box';
+import { useAccordion } from '../hooks/useAccordion';
+
+import { StyledAccordion, StyledAccordionButton, StyledBox, StyledGridItem } from './styled';
+
+export interface AccordionPanelProps extends HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  header: string;
+  isExpanded: boolean;
+  onClick?: () => void;
+  iconLeft?: React.ReactNode;
+  defaultExpanded?: boolean;
+}
+
+export const AccordionPanel: React.FC<AccordionPanelProps & BoxProps> = memo(
+  ({ children, header, iconLeft, defaultExpanded = false, ...props }) => {
+    const { isExpanded, onClick } = useAccordion({ defaultExpanded });
+    const accordionId = useUniqueId('accordion');
+    const accordionItemId = useUniqueId('accordion-item');
+
+    return (
+      <StyledAccordion {...props} isExpanded={isExpanded}>
+        <StyledAccordionButton
+          aria-controls={accordionItemId}
+          aria-expanded={isExpanded}
+          id={accordionId}
+          onClick={onClick}
+          variant="subtle"
+        >
+          <StyledGridItem>
+            {iconLeft}
+            <Text as="span" marginLeft={iconLeft ? 'small' : 'none'}>
+              {header}
+            </Text>
+          </StyledGridItem>
+          <StyledGridItem>
+            {isExpanded ? (
```

Comment:
Maybe we can try and use the button props instead of trying to manually layout the icons?

Reply:
Sorry kinda late to the party here, but should these icons be `<Button iconOnly={...} />` ? Or is that what you mean?

Reply:
^ Yeah, was trying to suggest using the built-in `icon*` props instead of a custom grid. We can style the `StylableButton` so if we need to move things around we can update it in the styled-component rather than markup.